frontend/src/components/Log.js

```diff
@@ -1,23 +1,50 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect } from 'react';
 import LogEntry from './LogEntry';
-import { logEntryService } from '../services/logEntryService';
+import { connect } from 'react-redux';
+import { getTroopsAction } from '../actions/TroopsActions';
+import { getBuildingsAction } from '../actions/BuildingsActions';
 import './Log.css';
 
-const Log = () => {
-  const [data, setData] = useState([]);
+const Log = ({ kingdom, buildings, troops, get }) => {
 
   useEffect(() => {
-    setData(logEntryService.readLogEntries());
-  }, []);
+    get(kingdom);
```

Comment:
@gulyaasjaanos let's name this a little better :D
something like `fetchBuildingsAndTroops`